feature/bookmarks/src/main/java/com/google/samples/apps/nowinandroid/feature/bookmarks/BookmarksScreen.kt

```diff
@@ -102,18 +110,65 @@ fun BookmarksScreen(
                     item(span = { GridItemSpan(maxLineSpan) }) {
                         Spacer(Modifier.windowInsetsBottomHeight(WindowInsets.safeDrawing))
                     }
-                } else item(span = { GridItemSpan(maxLineSpan) }) {
-                    Box(
-                        modifier = Modifier
-                            .fillMaxHeight()
-                            .wrapContentSize()
-                            .testTag("bookmarks:empty"),
-                        contentAlignment = Alignment.Center
-                    ) {
-                        Text(text = stringResource(id = R.string.bookmarks_empty))
-                    }
                 }
             }
         }
     }
+
+    /**
+     ** The [Column] is handling the Empty state when the feed is empty.
+     **/
+    if (feedState is NewsFeedUiState.Success && feedState.feed.isEmpty()) {
+        OnEmptyStateView()
+    }
 }
+
+@Composable
+private fun OnEmptyStateView() {
+    Column(
+        modifier = Modifier
+            .padding(start = 16.dp, end = 16.dp)
+            .fillMaxSize()
+            .testTag("bookmarks:empty"),
+        verticalArrangement = Arrangement.Center,
+        horizontalAlignment = Alignment.CenterHorizontally,
+    ) {
+        Image(
+            modifier = Modifier
+                .fillMaxWidth(),
+            painter = painterResource(id = R.drawable.img_no_bookmarks_error),
+            contentDescription = stringResource(id = R.string.bookmarks_empty_description),
```

Comment:
Since you also add text, you can set the content description of the image to null (it's purely decorative)